services/core-api/app/api/mines/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -13,10 +13,14 @@
 
 from . import permit_amendment_status_code, permit_amendment_type_code
 from app.api.utils.models_mixins import AuditMixin, Base
+from app.api.constants import *
 
 
 class PermitAmendment(AuditMixin, Base):
     __tablename__ = 'permit_amendment'
+    _edit_groups = [PERMIT_EDIT_GROUP, PERMIT_AMENDMENT_EDIT_GROUP, MINE_EDIT_GROUP]
+    _edit_key = PERMIT_AMENDMENT_EDIT_GROUP
```

Comment:
Starting at permit amendments, you can only edit related entities with PERMIT_AMENDMENT_EDIT_GROUP in _edit_groups, in this example you cannot go up to edit a Permit from the amendment.